common/religion/religions/lizard_group.txt

```diff
@@ -24,6 +24,20 @@
 	doctrine = doctrine_deviancy_shunned
 	doctrine = doctrine_witchcraft_shunned
 
+	#Magic
+	doctrine = doctrine_light_magic_ignored
+	doctrine = doctrine_shadow_magic_crime
+	doctrine = doctrine_disorder_magic_crime
+	doctrine = doctrine_order_magic_ignored
+	doctrine = doctrine_life_magic_accepted
+	doctrine = doctrine_death_magic_crime
+	doctrine = doctrine_elemental_spirit_magic_shunned
```

Comment:
Well, they worship the giant dinosaur spirit.
https://wow.gamepedia.com/Kros

Reply:
Done?

Reply:
> Done?

I would make it so they accept Decay.

Reply:
Done